.travis.yml

```diff
@@ -1,7 +1,7 @@
 os: linux
 dist: xenial
 language: python
-python: 3.8.6
+python: 3.9.0
```

Comment:
Question: Is the intent to run "latest" python here, or "latest 3.9", or be specific to 3.9.0? If some form of latest is intended, should the specification be more general, such as "3.9"?

Reply:
For Travis, if you specify "3.9" it will pick a version, not necessarily the latest.  So if you want the latest, you need to specify it.  This is most important to get more recent versions of the packaged tools with that version of python, i.e. `setuptools` and `pip`.  They don't have a good way of indicating "install the latest 3.8 or 3.9".  But this is an idiosyncracy of Travis.  I'm completely deleting the Travis YAML file in this PR.

Reply:
Oh, and 3.9.0 is the only 3.9 currently.  So I was someone over-specifying it here, but better to do that than to leave it at "3.9" and have people think that when 3.9.1 comes out in a month or two, that it will somehow automatically be used.  It won't.